automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -0,0 +1,211 @@
+package org.broadinstitute.dsde.test.api
+
+import java.util.UUID
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import org.broadinstitute.dsde.workbench.config.{Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.fixture._
+import org.broadinstitute.dsde.workbench.service._
+import org.broadinstitute.dsde.workbench.util.Retry
+import org.broadinstitute.dsde.workbench.service.test.{CleanUp, RandomUtil}
+import org.scalatest.{FreeSpecLike, Matchers}
+import spray.json._
+
+class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with Matchers
+  with CleanUp with RandomUtil with Retry
+  with BillingFixtures with WorkspaceFixtures {
+
+  val Seq(studentA, studentB) = UserPool.chooseStudents(2)
+  val studentAToken: AuthToken = studentA.makeAuthToken()
+  val studentBToken: AuthToken = studentB.makeAuthToken()
+
+  val owner: Credentials = UserPool.chooseProjectOwner
+  val ownerAuthToken: AuthToken = owner.makeAuthToken()
+
+  "Rawls" - {
+    "should allow project owners" - {
+      "to create, clone, and delete workspaces" in {
+        implicit val token: AuthToken = ownerAuthToken
+
+        withCleanBillingProject(owner) { projectName =>
+          val workspaceName = s"${UUID.randomUUID().toString()}-owner-test-workspace"
```

Comment:
Since you're using `s"${UUID.randomUUID().toString()}-` a lot you could define a function for it